src/domain/data-decoder/entities/data-decoded.entity.ts

```diff
@@ -0,0 +1,11 @@
+export interface DataDecodedParameter {
+  name: string;
+  param_type: string;
+  value: string | number;
+  value_decoded?: Record<string, any>;
```

Comment:
Shouldn't this be `valueDecoded`?

Reply:
Hmm, yes I'm seeing `valueDecoded` now as well as part of the TX service responses. But I think I copied this from another response... not sure. Due to the misalignment of names, this wasn't being actually validated. I adapted the schema to accept both an object and an array in the `valueDecoded` field (I'm seeing an array there for a multiSend transaction), but as I commented I lack of the knowledge to define a more fine-grained schema here. 

The change is on [34fb5c7ad694e4f957b0d3b7c748cf27c0cac630](https://github.com/5afe/safe-client-gateway-nest/pull/132/commits/34fb5c7ad694e4f957b0d3b7c748cf27c0cac630). I needed to define it as `Schema` instead of `JsonSchemaType`. Maybe another option could be just define the required properties only (`name` and `value`) in `dataDecodedParameterSchema` to have a more flexible schema, what do you think?

Reply:
I didn't found a better solution for now, so I kept the `Schema` definition. We may revisit this in the future to try to get all the benefits from the TS/AJS integration 🙂